app/src/main/java/com/example/pix/chat/ChatActivity.java

```diff
@@ -10,19 +10,25 @@
 import android.os.Bundle;
 
 import com.example.pix.R;
+import com.example.pix.home.fragments.ComposeFragment;
 
 public class ChatActivity extends AppCompatActivity {
 
-    public static final int RESULT_LOAD_IMG = 100;
-    public static final int REQUEST_PERM = 101;
+    public static final String NEW_PIC_CODE = "newPic";
```

Comment:
Check out the Android documentation on how to name these keys when building an intent. https://developer.android.com/guide/components/intents-filters 
I think these constants should be declared in the class that the intent was sent, and referenced in everywhere else. So in this case, put it in SearchAdapter.